test/models/episode_story_handler_test.rb

```diff
@@ -32,5 +32,52 @@
     first_audio.must_equal 's3://mediajoint.production.prx.org/public/audio_files/1200648/lcs_spring16_act1.mp3'
     last_audio.must_equal 's3://mediajoint.production.prx.org/public/audio_files/1200657/broadcast/t01.mp3'
     episode.description.must_equal 'this is a description'
+    episode.season_number.must_equal 2
+    episode.episode_number.must_equal 4
+    episode.clean_title.must_equal 'Stripped-down title'
   end
+
+  describe 'with episode identifiers' do
+
+    let(:zero_identifiers_story) do
+      msg = json_file(:prx_story_zero_identifiers)
+      body = JSON.parse(msg)
+      href = body['_links']['self']['href']
```

Comment:
I just started using this `dig` method, and it is life changing
https://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig

Reply:
ooh boy. new favorite